source/standards/technical-operations.html.md.erb

```diff
@@ -1,10 +1,33 @@
 ---
-title: Cyber Security team
-last_reviewed_on: 2020-02-05
+title: Technical Operations
+last_reviewed_on: 2020-02-17
 review_in: 6 months
 ---
 
-# <%= current_page.data.title %>
+# Technical operations team
+
+The Technical operations team focuses on improving how we run services and make decisions about technology while supporting the other service teams within GDS.
+
+We work with GDS technology teams to:
+
+* provide a common set of tools to support common problems
+* integrate and configure those tools
+* build and maintain a secure platform making it easy for teams to build services
+* provide Security Operations, Engineering and Architecture support to other teams in GDS
```

Comment:
I think we should probably include user support in this, if we're going to expand the RE / Cyber section into the whole of techops.

```suggestion
* provide Security Operations, Engineering and Architecture support to other teams in GDS
* provide a first line support function for end users - engaging with them, understanding their problems, and helping resolve them
```

(copy stolen from the techops weeknote)

Reply:
Discussed offline and we'll save sorting this out for another later PR :)